tests/transactions/tests.py

```diff
@@ -443,8 +443,25 @@ def test_atomic_does_not_leak_savepoints_on_failure(self):
                 # This is expected to fail because the savepoint no longer exists.
                 connection.savepoint_rollback(sid)
 
-    @skipIf(connection.features.autocommits_when_autocommit_is_off,
-            "This test requires a non-autocommit mode that doesn't autocommit.")
+
+@skipIf(
+    connection.features.autocommits_when_autocommit_is_off,
+    "This test requires a non-autocommit mode that doesn't autocommit."
+)
+class NonAutocommitTests(TransactionTestCase):
+
+    available_apps = []
+
+    def test_orm_query_after_error(self):
```

Comment:
I just ran my test in 1.6, 1.7, 1.8, and 1.9. This regression was first added in 1.9 (https://bitbucket.org/marky1991/django-test/raw/59c9ff89e4b12b4a831c36171139cb022735201b/test1.py first fails in 1.9). We didn't notice then because we are updating from 1.7 to 1.10. Non-autocommit mode has definitely worked fine up until 1.7 (we use it everywhere in our code and we use 1.7 in production).

In any case, we have an internal monkeypatch workaround and can wait to remove that until 1.11 . I will work on reponding to the review so we can close this ticket.